src/components/home-screen.vue

```diff
@@ -140,16 +139,26 @@ export default {
       'showHotWater',
       'showHumidity'
     ]),
-    boostEnabled() {
-      const modeState = this.$store.state.modes[this.selectedMode]
-      return modeState && modeState.boostEnabled
-    },
-    boostTimeRemaining() {
-      const modeState = this.$store.state.modes[this.selectedMode]
-      if (modeState) {
-        return modeState.boostTimeRemaining
+    powerSettingText() {
+      const modes = {
+        cool: () => 'Cooling',
+        heat: () => this.modes.heat2.running ? '2nd-stage heating' : 'Heating',
+        hotwater: () => 'Hot water',
+        humidity: () => 'Humidity control',
+        fan: () => 'Fan'
       }
-      return null
+      if (this.selectedMode && modes[this.selectedMode]) {
+        const modeState = this.modes[this.selectedMode]
+        const modeText = modes[this.selectedMode]()
+        if (modeState.boostEnabled) {
+          return `${modeText} boost mode, ${modeState.boostTimeRemaining} min. remaining`
+        }
+        if (modeState.active) {
```

Comment:
Selecting fan ON, shows 'Fan auto'.
Selecting AUTO shows no change which is the expected behaviour, so there is no such thing as Fan auto.
Can you differentiate this return according to slectedMode?

Reply:
Is there a reason you can even select "Auto" then? Auto should be forced by heating and cooling as needed. The user should only need to select between "On" and "Off".

Reply:
In theory no but if someone is not using heating/cooling and just wants fan for ventilation the ON/OFF options seem more straightforward rather than ON/AUTO. Auto is mostly something that will be selected by OH if in OFF. Do you think it is confusing? I spied a bit on commercial units for that...

Reply:
Just re-read your comment. We say the same thing... Hiding Auto will mess things up in the backend I think. Let's not remove it yet. I opened an issue on me for this. Do you want to leave Auto on the LCD too for consistency with the OH UI?

Reply:
If you prefer the consistency then let's resolve the conversation over in #9 and whatever decision is reached about the OH UI we will reflect it in the touch UI (spoilers: I usually prefer the UX that involces less things on the screen and less choices :slightly_smiling_face: )

Reply:
Following my comment on #9 I would suggest to simplify the Touch UI only and leave an open issue for OH UI for Fan.